### PR TITLE
Add a CRA example app

### PR DESCRIPTION
**Pull request by HichamELBSI**
This is just a CRA example to test the DS build and components.

Signed-off-by: HichamELBSI <elabbassih@gmail.com>

### CODE REVIEW

**Comment by HichamELBSI** [comment]
This is just a CRA example to test the DS build and components.

File: examples/app/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (soupette) [inline examples/app/README.md:1]: Maybe we could change the README a bit?

File: examples/app/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@buffetjs/core": "^3.3.5",
```
Comment (soupette) [inline examples/app/package.json:6]: Do you plan on removing them later?

File: examples/app/public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```
Comment (soupette) [inline examples/app/public/index.html:12]: Maybe we could change the favicon to use the strapi one? (Not needed)

File: examples/app/public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```
Comment (soupette) [inline examples/app/public/manifest.json:2]: Can we change this so it is Strapi branded?

File: examples/app/src/App.css [hunk 5]
```diff
@@ -0,0 +1,37 @@
+.App {
```
Comment (soupette) [inline examples/app/src/App.css:1]: Not sure this is needed, WDYT?

File: examples/app/src/App.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import { Checkbox, Box } from "@strapi/design-system";
+import { Checkbox as BuffetCheckbox } from "@buffetjs/core";
+import "./App.css";
+
+function App() {
+  const [val, setValue] = useState();
```
Comment (soupette) [inline examples/app/src/App.js:7]: It's a nice start, but what do you think later to create a more dynamic form so it is easier to test the other input components?

File: examples/app/src/index.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+import { ThemeProvider, lightTheme } from "@strapi/design-system";
+import { GlobalStyle } from "@buffetjs/styles";
+import "./index.css";
+import Main from "./App";
+import reportWebVitals from "./reportWebVitals";
+
+const App = () => {
+  return (
+    <ThemeProvider theme={lightTheme}>
+      <GlobalStyle />
```
Comment (soupette) [inline examples/app/src/index.js:12]: Maybe you can create 2 pages, one for buffet one for the new DS so we can easily see the difference between them?

File: package.json [hunk 8]
```diff
@@ -20,6 +20,7 @@
     "specify:pull": "specify pull"
   },
   "devDependencies": {
+    "@craco/craco": "^6.1.2",
```
Comment (soupette) [inline package.json:23]: This dependency needs to be in the examples/app so if we choose to remove the exemple app we don't have to remember to clean the repo dependencies.

Comment (soupette) [reply package.json:23]: Also what do you think about create a new script:

`setup: "yarn && yarn bootstrap && yarn build"` so we have only command that does everything?

File: yarn-error.log [hunk 9]
```diff
@@ -0,0 +1,6813 @@
+Arguments:
```
Comment (soupette) [inline yarn-error.log:1]: Maybe we should add this file to the .gitignore WDYT?

**Review by soupette — COMMENTED** [review]
Good job we can clearly see the difference between the 2 component libraries!

File: examples/app/src/App.css (hunk 5 above)
Comment (HichamELBSI) [reply examples/app/src/App.css:1]: Indeed, it is not :)

File: examples/app/src/App.js (hunk 6 above)
Comment (HichamELBSI) [reply examples/app/src/App.js:7]: Yes, I will do it when we will have more components. Not sure it is relevant to do it actually with a checkbox and radio only

File: examples/app/src/index.js (hunk 7 above)
Comment (HichamELBSI) [reply examples/app/src/index.js:12]: Not sure it is relevant to keep buffet in this app. I will remove it.

File: package.json (hunk 8 above)
Comment (HichamELBSI) [reply package.json:23]: Why not 😄

File: yarn-error.log (hunk 9 above)
Comment (HichamELBSI) [reply yarn-error.log:1]: yep

**Review by soupette — APPROVED** [review]
LGTM this is awesome! 👏

**Pull request merged**